Auth0/CredentialsManager.swift

```diff
@@ -84,10 +84,37 @@ public struct CredentialsManager {
     /// Clear credentials stored in keychain
     ///
     /// - Returns: if credentials were removed
+    @available(*, deprecated, message: "use clearAndRevokeToken(callback) instead")
```

Comment:
In retrospect I'm not sure if this should be deprecated, there is nothing technically wrong with it as you don't have to use refresh tokens and can simply use it to securely store an AT, although that would be a less common use case.

Reply:
Still having the `clear()` function would certainly make the test suite cleaner; I've had to refactor them out to remove the compiler warnings, but doing that requires a sprinkling of async handling code throughout the tests.

Then it's just communicating to the user best practises around whether to use `clear()` or `revoke()` depending on their situation.

Reply:
I've removed the deprecation on this